kodi_addon_checker/check_addon.py

```diff
@@ -337,3 +341,58 @@ def number_of_lines(filepath):
         data = file.read()
 
     return (analyze(data).lloc)
+
+
+def _logProblem(error_counter, problem_string):
```

Comment:
I suspect a rebase error here. THis and the next method are not supposed to be used anymore.